server/lib/bitbucket.js

```diff
@@ -205,21 +208,21 @@ const getConnectionTreeByPath = (params, filePath) =>
 const getConnectionsTree = (params) =>
   new Promise((resolve, reject) => {
     try {
-      bitbucket().get(`repositories/{username}/{repo_slug}/src/{revision}/${constants.DATABASE_CONNECTIONS_DIRECTORY}`, params, (err, res) => {
+      bitbucket().get(`repositories/{username}/{repo_slug}/src/{revision}/${constants.DATABASE_CONNECTIONS_DIRECTORY}?pagelen=100`, params, (err, res) => {
```

Comment:
Although it's unlikely, what happens in the case we have > 100 connections. Do we need to include pagination as well?